tests/test_textures.py

```diff
@@ -0,0 +1,93 @@
+import os
+
+from pytest import fixture, skip
+from testutils import can_use_wgpu_lib
+
+
+if not can_use_wgpu_lib:
+    skip("Skipping tests that need the wgpu lib", allow_module_level=True)
+
+
+@fixture(autouse=True, scope="module")
+def force_offscreen():
+    os.environ["WGPU_FORCE_OFFSCREEN"] = "true"
+    try:
+        yield
+    finally:
+        del os.environ["WGPU_FORCE_OFFSCREEN"]
```

Comment:
You could move this fixture to conftest.py so it is used across the test suite.

Reply:
okay, also removed it from the other test scripts. Test are still passing.